demisto_sdk/commands/pre_commit/hooks/docker.py

```diff
@@ -171,8 +178,9 @@ def _split_by_objects(
     ] = defaultdict(set)
 
     for file, obj in files_with_objects:
-        if split_by_obj or (config_arg and (obj.path.parent / config_arg[1]).exists()):
+        if run_isolated or (config_arg and (obj.path.parent / config_arg[1]).exists()):
```

Comment:
extract `confiug_arg[1]` to a variable called `file_name`, would be much more readable IMO